### PR TITLE
Feature/vault sub nav

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Adds vault-sub-nav and use-vault-sub-nav
- Fixes other style issues

### CODE REVIEW

File: apps/web/src/index.html [hunk 1]
```diff
@@ -35,6 +35,7 @@
       width: 100%;
       overflow: scroll !important;
       overflow-x: auto !important;
+      margin: 0;
```
Comment (jeffywu) [inline apps/web/src/index.html:38]: This is to fix the jumping issue?

Comment (matthew-garrett) [reply apps/web/src/index.html:38]: No, this was a mistake. I'll fix.

File: packages/features/vault/src/components/vault-sub-nav/use-vault-sub-nav.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    {
+      label: 'Back to top',
+      key: VAULT_SUB_NAV_ACTIONS.BACK_TO_TOP,
+      callback: () => scroll.scrollToTop(),
+    },
+    {
+      label: 'Full Documentation',
+      key: VAULT_SUB_NAV_ACTIONS.FULL_DOCUMENTATION,
+      href: 'https://docs.notional.finance/leveraged-vaults/leveraged-vaults/balancer-aura-wsteth-weth-strategy',
```
Comment (jeffywu) [inline packages/features/vault/src/components/vault-sub-nav/use-vault-sub-nav.tsx:34]: This is likely going to have to change to a prop that gets passed in later, I guess we need to figure out how to get the right link in here...we can leave it as is for now

File: packages/features/vault/src/components/vault-sub-nav/use-vault-sub-nav.tsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { animateScroll as scroll } from 'react-scroll';
+import { VAULT_SUB_NAV_ACTIONS } from '@notional-finance/shared-config';
+
+export const useVaultSubNav = () => {
+  const subNavData = [
+    {
+      label: 'Overview',
```
Comment (jeffywu) [inline packages/features/vault/src/components/vault-sub-nav/use-vault-sub-nav.tsx:7]: Labels probably should be I18N compatible

File: packages/features/vault/src/components/vault-sub-nav/vault-sub-nav.tsx [hunk 4]
```diff
@@ -1,5 +1,180 @@
...
+  }
+`
+);
+
+const NavItemContainer = styled(Box)(
+  ({ theme }) => `
+  flex: 1;
+  display: flex;
+  min-width: ${theme.spacing(94)};
+  padding-right: -15px;
```
Comment (jeffywu) [inline packages/features/vault/src/components/vault-sub-nav/vault-sub-nav.tsx:153]: Use theme.spacing here and on the max-width below

File: packages/features/vault/src/components/vault-sub-nav/vault-sub-nav.tsx [hunk 5]
```diff
@@ -1,5 +1,180 @@
...
+  }
+  .active {
+    height: 100%;
+    background: ${theme.palette.info.light};
+    color: ${reverseTheme.palette.typography.accent};
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-weight: 600;
```
Comment (jeffywu) [inline packages/features/vault/src/components/vault-sub-nav/vault-sub-nav.tsx:126]: Can we use one of the font weights from the theme?

**Review by jeffywu — APPROVED** [review]

**Review by jeffywu — APPROVED** [review]

**Pull request merged**